src/io/flutter/inspector/InspectorService.java

```diff
@@ -112,6 +110,26 @@ public void addClient(InspectorServiceClient client) {
     return getInspectorLibrary().eval("WidgetInspectorService.instance." + methodName + "(arg1, \"" + groupName + "\")", scope);
   }
 
+  public CompletableFuture<Void> setPubRootDirectories(List<String> rootDirectories) {
+    // TODO(jacobr): remove call to hasServiceMethod("setPubRootDirectories") after
+    // the `setPubRootDirectories` method has been in two revs of the Flutter Alpha
+    // channel. The feature is expected to have landed in the Flutter dev
+    // chanel on March 2, 2018.
+
+    return hasServiceMethod("setPubRootDirectories").thenComposeAsync((Boolean hasMethod) -> {
+      if (!hasMethod) {
+        return null;
+      }
+      JsonArray jsonArray = new JsonArray(rootDirectories.size());
```

Comment:
It looks like this is breaking some build configs on travis:

```
     [javac2] /home/travis/build/flutter/flutter-intellij/src/io/flutter/inspector/InspectorService.java:123: error: constructor JsonArray in class JsonArray cannot be applied to given types;
     [javac2]       JsonArray jsonArray = new JsonArray(rootDirectories.size());
     [javac2]                             ^
     [javac2]   required: no arguments
```

I suppose we're getting different versions of the gson bits?

Maybe just use the default constructor here to work around?